### PR TITLE
Mesos operations metrics

### PR DESCRIPTION
**Pull request by philipnrmn**
## High-level description

This PR adds [mesos operations metrics](https://mesos.apache.org/documentation/latest/monitoring/#operations) to the Telegraf metrics pipeline.

## Corresponding DC/OS tickets (required)

  - [DCOS_OSS-5023](https://jira.mesosphere.com/browse/DCOS_OSS-5023) Add Mesos operations metrics to Telegraf

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/telegraf/compare/4af92b507da53dd1d06a6637544b97b04724e5f0...d86594f3e17361973c41e0de015cbc110ec3b2a1)
  - [x] Included a test which will fail if code is reverted but test is not. 
  - [x] Test Results: [link to CI job test results for component](https://jenkins.mesosphere.com/service/jenkins/job/public-dcos-cluster-ops/job/telegraf/job/telegraf-dcos-pulls/264/)
  - [x] Code Coverage (if available): [link to code coverage report](https://jenkins.mesosphere.com/service/jenkins/job/public-dcos-cluster-ops/job/telegraf/job/telegraf-dcos-pulls/264/)

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 ### What's new
 
+* Updated telegraf to process mesos operations metrics (DCOS_OSS-5023)
+
+* Upgraded CockroachDB to version [2.1.7](https://www.cockroachlabs.com/docs/releases/v2.1.7.html). (DCOS_OSS-5262)
```
Comment (jgehrcke) [inline CHANGES.md]: Oh, that line got in here by accident, right?

Comment (jgehrcke) [reply CHANGES.md]: Let me add a commit and remove it.

**Comment by jgehrcke** [comment]
Added two commits for CHANGES.md

**Comment by jgehrcke** [comment]
@philipnrmn wanna put this into `ready for review`?

**Comment by philipnrmn** [comment]
JP, the upstream PR just merged, it's ready now :)

**Comment by philipnrmn** [comment]
@mesosphere-mergebot label ready for review

**Comment by philipnrmn** [comment]
@gracedo and @branden would you kindly review this PR?

**Comment by philipnrmn** [comment]
@mesosphere-mergebot request-review @branden @gracedo

File: CHANGES.md (hunk 1 above)
Comment (gracedo) [reply CHANGES.md]: does this line still need to be removed?

Comment (philipnrmn) [reply CHANGES.md]: I rebased and it came back :sleepy:

Comment (philipnrmn) [reply CHANGES.md]: OK, it's gone for real now.

File: CHANGES.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 ### What's new
 
+* Updated telegraf to process mesos operations metrics (DCOS_OSS-5023)
```
Comment (jgehrcke) [inline CHANGES.md]: Next time let's capitalize Telegraf and Mesos and add a full stop please (-:

**Pull request merged**